tests/rundoctest.py

```diff
@@ -8,12 +8,18 @@
 path = os.path.dirname(__file__)
 if path:
     os.chdir(path)
+os.chdir('..')
```

Comment:
Why do you want to use chdir? Isn't it safer to run the tests from the repository tests/? I was proposing
```
path = os.path.dirname(__file__)
if path:
    os.chdir(path)
sys.path.append(os.path.abspath(os.path.join(path, os.path.pardir)))
```